src/tmpfiles/tmpfiles.c

```diff
@@ -702,6 +717,8 @@ static int parse_xattrs_from_arg(Item *i) {
                         break;
 
                 r = specifier_printf(xattr, specifier_table, NULL, &xattr_replaced);
+                if (r == -ENOKEY)
+                        return r;
                 if (r < 0)
```

Comment:
hmm, are you resolving specifiers twice on the xattrs now? I figure this oinvocation of specifier_printf() should be dropped, no?

Reply:
good catch !